### ISSUE TITLE
issue when using symfony HttpCache cache

### ISSUE THREAD

**Issue by igorRovenki** [FriendsOfSymfony/FOSHttpCacheBundle#193]
hi guys!
I have no varnish or nginx, I just trying to use symfony proxy cache with symfony built-in server and getting this error:
```
Fatal error: Uncaught exception 'Symfony\Component\DependencyInjection\Exception\RuntimeException' with message 'The parent definition "fos_http_cache.request_matcher" defined for definition "fos_http_cache.request_matcher.ea0361456f2444f1464abc61f1decd4b4215e2d10d07225889f2fba40bca65edd3622edf" does not exist.' in /Users/igor/PhpstormProjects/site.com/vendor/symfony/symfony/src/Symfony/Component/DependencyInjection/Compiler/ResolveDefinitionTemplatesPass.php:67 Stack trace: #0 /Users/igor/PhpstormProjects/site.com/vendor/symfony/symfony/src/Symfony/Component/DependencyInjection/Compiler/ResolveDefinitionTemplatesPass.php(50): Symfony\Component\DependencyInjection\Compiler\ResolveDefinitionTemplatesPass->resolveDefinition('fos_http_cache....', Object(Symfony\Component\DependencyInjection\DefinitionDecorator)) #1 /Users/igor/PhpstormProjects/site.com/vendor/symfony/symfony/src/Symfony/Component/DependencyInjection/Compiler/Compiler.php(119): Symfony\Component\DependencyInjection\Compiler\ResolveDefinitionTemplates in /Users/igor/PhpstormProjects/site.com/vendor/symfony/symfony/src/Symfony/Component/DependencyInjection/Compiler/ResolveDefinitionTemplatesPass.php on line 67
503 Service Unavailable
```
```yaml
#app/config/config.yml
fos_http_cache:
    proxy_client:
        varnish:
            servers: 127.0.0.1:8005
            base_url: localhost:8005
    invalidation:
        enabled: true    # Defaults to 'auto'
        rules:
            -
                match:
                    attributes:
                        _route: "admin_archive_work_edit|admin_archive_work_delete"
                routes:
                    default_media_index: ~
```
```php
// app/AppCache.php
require_once __DIR__.'/AppKernel.php';

use FOS\HttpCacheBundle\SymfonyCache\EventDispatchingHttpCache;
use Symfony\Bundle\FrameworkBundle\HttpCache\HttpCache;
use Symfony\Component\HttpFoundation\Request;
use Symfony\Component\HttpFoundation\Response;

class AppCache extends EventDispatchingHttpCache
{
    protected function invalidate(Request $request, $catch = false)
    {
        if ('PURGE' !== $request->getMethod()) {
            return parent::invalidate($request, $catch);
        }

        if ('127.0.0.1' !== $request->getClientIp()) {
            return new Response('Invalid HTTP method', Response::$statusTexts[400]);
        }

        $response = new Response();
        if ($this->getStore()->purge($request->getUri())) {
            $response->setStatusCode(200, 'Purged');
        } else {
            $response->setStatusCode(200, 'Not found');
        }

        return $response;
    }
}
```
What's I'm doing wrong?

**Comment by ddeboer** [comment]
Are you trying to invalidate (purge) URLs with Symfony HttpCache? Our bundle doesn’t yet support that: keep an eye on #168 to see when we finish it.

**Comment by igorRovenki** [comment]
@ddeboer, yea I'm want to purge with Symfony HttpCache, thanks for response

**Comment by ddeboer** [comment]
Fixed and tagged 1.2.1.
